front/app/containers/IdeasNewPage/index.tsx

```diff
@@ -33,9 +32,11 @@ const IdeasNewPage = () => {
   } = useProjectBySlug(slug);
   const { data: authUser } = useAuthUser();
   const { data: phases, status: phasesStatus } = usePhases(project?.data.id);
-  const { phase_id } = parse(location.search, {
-    ignoreQueryPrefix: true,
-  }) as { [key: string]: string };
+  const [searchParams] = useSearchParams();
+  // If we reach this component by hitting ideas/new directly, without a phase_id,
+  // we'll still get to this component, so we try to get the phase id from getCurrentPhase.
+  const phaseId =
+    searchParams.get('phase_id') || getCurrentPhase(phases?.data)?.id;
```

Comment:
This should also increase safety. I.e., if we link to ideas/new or surveys/new somewhere, without adding `phase_id` to the search params, we have this fallback. I think we need a better solution for the `phase_id` search param though so added it to the FE CDM agenda.